plugins/kapt3/kapt3-base/src/org/jetbrains/kotlin/kapt3/base/ProcessorLoader.kt

```diff
@@ -100,7 +100,7 @@ open class ProcessorLoader(private val options: KaptOptions, private val logger:
                         processSingleInput(it.inputStream())
                     }
                 }
-                file.isFile -> {
+                file.isFile && file.extension == "jar" -> {
```

Comment:
Maybe we should check it case-insensitively?

Reply:
Good catch, updated in 627380798c8026d6625e4edc3a669a6dbbae2fc5.